18-dualstack_ipv4_ipv6_seed_shoot.md

```diff
@@ -27,7 +27,23 @@ The easiest solution is to provide both protocol stacks - IPv4/IPv6 dual stack.
 
 ### Configuration
 
-> TODO
+In the shoot specification we will use the definition of the `network` section to provide the cidrs for `nodes`, `pods` and `services`. As those fields already exist they will be enhanced to accept a tuple of values. One value for IPv4 and one for IPv6. We introduce special key words to request different strategies for choosing the respective CIDRs. These will be available at runtime only and being configured by Gardener/K8S during the cluster deployment/reconciliation.
+
+The following keywords are currently anticipated
+- **`autoV4`/`autoV6`** use the cloud provider's IPAM to reserve a prefix for the range (optional: of at least that size).
+- **`noV4`/`noV6`** disable IPv4/IPv6 support
+- **`+nodePrefix`** modifyer – request a prefix per node (optional: of at least that size)
```

Comment:
Well, isn't this dependent on the cloud provider? For IPv4 this also depends on the CNI provider.

Reply:
It will be… but at least for AWS, there is some flexibility (between /64 and /80)